# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 14eab025027a
# Full Hash: 14eab025027a09c77de45e0d0c910b2a37eec082
# Author: Timothy Nikkel <tnikkel@gmail.com>
# Date: 2021-03-12 15:32:35
# Regressor Bug: 1696802
# File Overlap Count: 1
# Description:
#   Bug 1696802. Add code to support double tap to zoom on mac. r=botond,mstange
#   
#   We already have a pref for double tap to zoom with is already enabled by default so it's kind of awkward to add another pref.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D107389
# ==============================================================================

diff -r 759586ace266 -r 14eab025027a modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Fri Mar 12 01:46:25 2021 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Fri Mar 12 02:10:36 2021 +0000
@@ -278,6 +278,11 @@
   value: true
   mirror: always
 
+- name: apz.mac.enable_double_tap_zoom_touchpad_gesture
+  type: RelaxedAtomicBool
+  value: false
+  mirror: always
+
 - name: apz.allow_immediate_handoff
   type: RelaxedAtomicBool
   value: false